server/spec/serializers/rpc/service_pod_serializer_spec.rb

```diff
@@ -175,6 +175,24 @@
       end
     end
 
+    describe '[:secrets]' do
+      it 'includes certificates as secrets' do
+        Certificate.create!(grid: grid,
+          subject: 'kontena.io',
+          valid_until: Time.now + 90.days,
+          private_key: 'private_key',
+          certificate: 'certificate',
+          full_chain: 'full_chain')
```

Comment:
```
  1) Rpc::ServicePodSerializer#to_hash [:secrets] includes certificates as secrets

     Failure/Error: Certificate.create!(grid: grid,

     Mongoid::Errors::UnknownAttribute:

       

       message:

         Attempted to set a value for 'full_chain' which is not allowed on the model Certificate.

       summary:

         Without including Mongoid::Attributes::Dynamic in your model and the attribute does not already exist in the attributes hash, attempting to call Certificate#full_chain= for it is not allowed. This is also triggered by passing the attribute to any method that accepts an attributes hash, and is raised instead of getting a NoMethodError.

       resolution:

         You can include Mongoid::Attributes::Dynamic if you expect to be writing values for undefined fields often.

     # ./vendor/bundle/ruby/2.4.0/gems/mongoid-5.2.1/lib/mongoid/attributes/processing.rb:99:in `process_attribute'

     # ./vendor/bundle/ruby/2.4.0/gems/mongoid-5.2.1/lib/mongoid/attributes/processing.rb:25:in `block in process_attributes'

     # ./vendor/bundle/ruby/2.4.0/gems/mongoid-5.2.1/lib/mongoid/attributes/processing.rb:23:in `each_pair'

     # ./vendor/bundle/ruby/2.4.0/gems/mongoid-5.2.1/lib/mongoid/attributes/processing.rb:23:in `process_attributes'

     # ./vendor/bundle/ruby/2.4.0/gems/mongoid-5.2.1/lib/mongoid/document.rb:111:in `block in initialize'

     # ./vendor/bundle/ruby/2.4.0/gems/mongoid-5.2.1/lib/mongoid/threaded/lifecycle.rb:89:in `_building'

     # ./vendor/bundle/ruby/2.4.0/gems/mongoid-5.2.1/lib/mongoid/document.rb:105:in `initialize'

     # ./vendor/bundle/ruby/2.4.0/gems/mongoid-5.2.1/lib/mongoid/persistable/creatable.rb:179:in `new'

     # ./vendor/bundle/ruby/2.4.0/gems/mongoid-5.2.1/lib/mongoid/persistable/creatable.rb:179:in `block in create!'

     # ./vendor/bundle/ruby/2.4.0/gems/mongoid-5.2.1/lib/mongoid/threaded/lifecycle.rb:161:in `_creating'

     # ./vendor/bundle/ruby/2.4.0/gems/mongoid-5.2.1/lib/mongoid/persistable/creatable.rb:175:in `create!'

     # ./spec/serializers/rpc/service_pod_serializer_spec.rb:180:in `block (4 levels) in <top (required)>'
```